kotlin/android/app/src/main/java/dev/firezone/android/features/settings/ui/SettingsFragment.kt

```diff
@@ -7,27 +7,28 @@ import android.view.inputmethod.EditorInfo
 import androidx.core.widget.doOnTextChanged
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.viewModels
+import androidx.navigation.fragment.findNavController
 import dev.firezone.android.R
-import dev.firezone.android.databinding.FragmentOnboardingBinding
+import dev.firezone.android.databinding.FragmentSettingsBinding
 import dagger.hilt.android.AndroidEntryPoint
 import dev.firezone.android.BuildConfig
 import dev.firezone.android.features.auth.ui.AuthActivity
 
 @AndroidEntryPoint
-internal class OnboardingFragment : Fragment(R.layout.fragment_onboarding) {
+internal class SettingsFragment : Fragment(R.layout.fragment_settings) {
```

Comment:
@jamilbk I was implementing a separate SettingsFragment, as it would have some UI for the user to delete the token and in debug mode a UI for us to view app logs.

Reply:
Ah, but wouldn't `Sign out` accomplish deleting the token? Or wiping the app's data? Confused why that would be in a separate view.

A debug log viewer in the SettingsFragment sounds good. Do you want to merge this and add your changes into it, or the other way around?

The main thing here was renaming `OnboardingFragment` since that's not really what this view is.

Reply:
I want to move the `Sign out` button to the settings screen too. We can merge this PR, I will push a separate PR for it.

Reply:
Sounds good. This is one is already large enough.